src/test/tables/tpch/minimal/customer.tbl

```diff
@@ -0,0 +1,3 @@
+c_custkey|c_name|c_address|c_nationkey|c_phone|c_acctbal|c_mktsegment|c_comment
+int|string|string|int|string|float|string|string
+0|Max Mustermann|Berlin|0|555-123-4567|10.0|BUILDING|-
```

Comment:
How would we have to choose the scaling factor so that we have at least two customers here?

Reply:
this is not a table generated by dbgen at all. If you want to replace the tables in `minimal/`, let's discuss this as part of another issue.

Reply:
ok